src/reports/components/assessment-report-summary.scss

```diff
@@ -24,6 +24,5 @@
         line-height: 24px;
         color: $primary-text;
         font-family: $font-family;
-        margin: 16px 0 21px 0;
```

Comment:
In general, I'd prefer to avoid relying on nonzero browser defaults - it makes it a lot harder for a future reader to understand whether a given bit of margin is intentional or not and to track down history

Reply:
That's a fair point, I added back the margin in [c7cf0dd](https://github.com/microsoft/accessibility-insights-web/pull/5727/commits/c7cf0dd0b34e7d6ec71576cbc0cc0c93df61593c).